packages/app/hooks/chat/useChat.ts

```diff
@@ -34,10 +34,7 @@ export const useChat = (itemTypeId = null) => {
 
   const [userInput, setUserInput] = useState('');
 
-  const { data: chatsData, refetch } = useGetUserChats(
-    user.id,
-    typeId.itemTypeId,
-  );
+  const { data: chatsData, refetch } = useGetUserChats(user.id, typeId);
```

Comment:
typeId is an object with a property itemTypeId, so this will fail to fetch the user chat I believe.

Reply:
You can check this pack: https://packrat.pages.dev/pack/jfxuhkd3zflt4ddavnyhh3so from testing preview, it has user chats but your PR preview: https://73a8f0cc.packrat.pages.dev/pack/jfxuhkd3zflt4ddavnyhh3so for the same pack doesn't. That might be why

Reply:
yeah you are right i forgot to add .itemTypeId after typeId